ts/utils/hooks/bottomSheet.tsx

```diff
@@ -168,3 +176,69 @@ export const useIOBottomSheetModal = (
   );
   return { present, dismiss: dismissAll, bottomSheet };
 };
+
+const DEFAULT_AUTORESIZABLE_SNAP_POINT = 1;
+const DEFAULT_BOTTOM_PADDING = 24;
```

Comment:
@dmnplb I've set the default bottom padding for the bottom sheet at 24 to calculate the automatic height, do we have any specific value for this in our design system configs?

Reply:
Thanks for the suggestion, [2077fa5](https://github.com/pagopa/io-app/pull/4631/commits/2077fa51eceb53f83c239e23b047f6487b9976e8) to implement!